SODA/Utilities/PositionsJsonConverter.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using SODA.Models;
+using System;
+
+namespace SODA.Utilities
+{
+    class PositionsJsonConverter : JsonConverter
+    {        
+        public override bool CanConvert(Type objectType)
+        {
+            return typeof(Positions).IsAssignableFrom(objectType);
+        }
+
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer) 
+        {
+            Positions positions = new Positions();
+            if (reader.TokenType == JsonToken.StartArray)
+            {
+                var arrayValues = JArray.Load(reader);
+
+                double arrayIndexOne = (double)arrayValues[0];
+                double arrayIndexTwo = (double)arrayValues[1];
+
+                positions.PositionsArray = new double[] { arrayIndexOne, arrayIndexTwo };
```

Comment:
I think we should prefer to use the `Positions` constructors whenever possible (rather than assign directly to `PositionsArray`), since we may be implementing logic to e.g. restrict the number of input elements in the array, etc. 

In fact we may just want to make `PositionsArray { get; private set; }` to enforce that.